R/ggplot-logo.R

```diff
@@ -0,0 +1,83 @@
+#' Add CorrelAid logo
+#'
+#' Inset CorrelAid logo to the bottom right corner of a ggplot.
+#'
+#' The size of the logo is determined by the base text size set e.g. by
+#'   the `base_size` argument in `theme_*()` functions.
+#'
+#' Adding the logo to a plot makes it a [`patchwork`][patchwork-package].
+#'   Please refer to the patchwork [documentation][patchwork-package] for
+#'   instructions on how to customise its appearance, e.g. via a theme or
+#'   scale function.
+#'
+#' @export
+#'
+#' @examples
+#' library(ggplot2)
+#'
+#' ggplot(mtcars, aes(wt, mpg)) + geom_point() + add_correlaid_logo()
+#'
+#' ggplot(example_projects_labels, aes(category)) +
+#'  geom_bar() +
+#'  labs(title = "Title", subtitle = "Subtitle", caption = "Caption") +
+#'  theme_correlaid() +
+#'  add_correlaid_logo()
+#'
+#' # Change colour scale after adding logo:
+#' ggplot(mtcars, aes(wt, mpg)) +
+#'   geom_point(size = 3) +
+#'   theme_correlaid(base_size = 20) +
+#'   add_correlaid_logo() &
```

Comment:
Because I use `patchwork::inset_element()` to add the logo, the original plot becomes a [patch in a patchwork](https://patchwork.data-imaginist.com/articles/guides/assembly.html#modifying-patches). Applying e.g. a theme to the entire patchwork using the `+` operator will not have any effect.

Will make it clearer in the documentation.